src/core/layout/qgslayoutitemelevationprofile.h

```diff
@@ -144,6 +144,20 @@ class CORE_EXPORT QgsLayoutItemElevationProfile: public QgsLayoutItem
      */
     double tolerance() const;
 
+    /**
+     * Returns whether the profile curve is set to follow the current atlas feature.
```

Comment:
@nyalldawson , we should expand the documentation here to add a note geometry type support (i.e. ATM line-only)